Purchases/Purchasing/PurchasesOrchestrator.swift

```diff
@@ -290,15 +290,24 @@ class PurchasesOrchestrator {
     @available(macOS, unavailable)
     @available(watchOS, unavailable)
     @available(tvOS, unavailable)
-    func beginRefundRequest(for productID: String, completion: @escaping (RefundRequestStatus, Error?) -> Void) {
-        beginRefundRequestHelper.beginRefundRequest(productID: productID) { result in
-            switch result {
-            case .failure(let error):
-                completion(.error, error)
-            case .success(let status):
-                completion(status, nil)
-            }
-        }
+    func beginRefundRequest(forProduct productID: String) async throws -> RefundRequestStatus {
```

Comment:
lint is saying `PurchasesOrchestrator` is too long now, but i saw a TODO on `purchase(package:` to "clean up, move to new class along with the private funcs below"...